### ISSUE TITLE
Show commit subject in monospace font

### ISSUE THREAD

**Issue by carun** [ForkIssues/TrackerWin#838]

**Comment by DanPristupov** [comment]
No, this option will not appear in the future.

**Comment by carun** [comment]
May I know why? At times I tend to introduce double spacing and having monospace font would help a lot.

**Comment by Eagle3386** [comment]
+1 for @carun - commit messages with monospace font (BTW: there's no option for customizing the font(s) at all, at least in Fork 1.50) would help a lot and since the Windows client uses WPF, it's surely a _very_ minor issue..

**Comment by DanPristupov** [comment]
@Eagle3386 there is an option in preferences to use a monospace font for commit description
<img width="567" alt="Screen Shot 2020-06-29 at 09 54 35" src="https://user-images.githubusercontent.com/618115/85987877-8b618300-b9ee-11ea-8e4e-533c7ccfa692.png">

Result:
<img width="600" alt="Screen Shot 2020-06-29 at 09 56 02" src="https://user-images.githubusercontent.com/618115/85988025-c06dd580-b9ee-11ea-852f-ae6accfd5bbc.png">

**Comment by Eagle3386** [comment]
@DanPristupov Yes, but it's simply "yes/no" instead of usual "which monospace font, which size, style, etc." and therefore neither what probably most users expect.
Additionally, your screenshot clearly shows what's missing - the description is monospace, but not the commit message itself - which is neither consistent (different font for description & message) nor aesthetic.

**Comment by carun** [comment]
@DanPristupov there's a misunderstanding. I am looking for monospace font when entering commit message, not when viewing the commit message.
![image](https://user-images.githubusercontent.com/471374/86027042-0de54500-b9e5-11ea-996a-01e474879f94.png)

**Comment by Eagle3386** [comment]
@DanPristupov Any news due to @carun's update?

**Comment by carun** [comment]
Hello @DanPristupov touching base to see if there's any update on this. It's been a year :-)
